composer.json

```diff
@@ -10,7 +10,8 @@
         }
     ],
     "require-dev": {
-        "composer/composer": "1.0.*@dev"
+        "composer/composer": "1.0.*@dev",
+        "cakephp/core": "dev-master"
```

Comment:
This is kind of annoying as it will always clone. We'll have to go to `3.*` once it exists.